docs/pages/connect-your-client/web-ui.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Using the Web UI
+description: Using the Teleport Web UI
+---
+The Teleport Web UI is a web-based visual interface from which you can access resources, view active sessions and recordings, create and review access requests, 
+manage users and roles, and more.
+
+This page serves a reference on certain Web UI features and their usage.
+
+## Joining an active session
+
+The Web UI allows you to list and join active SSH sessions via a web-based terminal. 
+
```

Comment:
I think it would be good to describe how to find a session to join. Otherwise, I'm not sure it's self-evident how to get here.

Reply:
To add to this point, we should also talk about what roles/permissions are required to see/join active sessions. I'm testing in my homelab, and my user with _all_ the roles doesn't see the active SSH session from my test user.